### PR TITLE
Update contributing.md

### PR DESCRIPTION
**Pull request by antoine2711**
Fixing malformed links.

Changes proposed in this pull request:
- Fixing 2 malformed links

### CODE REVIEW

**Review by wetneb — APPROVED** [review]
@antoine2711 can you check if you are able to merge this once someone else has approved?

**Comment by antoine2711** [comment]
> @antoine2711 can you check if you are able to merge this once someone else has approved?

@wetneb: 👍  Confirmed.

Thanks. Antoine

**Pull request merged**
